packages/pds/src/api/com/atproto/admin/updateAccountHandle.ts

```diff
@@ -27,7 +28,19 @@ export default function (server: Server, ctx: AppContext) {
           throw new InvalidRequestError(`Handle already taken: ${handle}`)
         }
       } else {
-        await ctx.plcClient.updateHandle(did, ctx.plcRotationKey, handle)
+        if (ctx.cfg.entryway) {
+          // the pds defers to the entryway for updating the handle in the user's did doc.
+          // here was just check that the handle is already bidirectionally confirmed.
+          // @TODO if handle is taken according to this PDS, should we force-update?
```

Comment:
i think probably yes because the entryway should be canonically deciding who gets which handle and the PDS should just inherit its decision